lib/vmdb/loggers/audit_logger.rb

```diff
@@ -1,13 +1,15 @@
-module Vmdb::Loggers
-  class AuditLogger < VMDBLogger
-    def success(msg)
-      info("Success") { msg }
-      $log.info("<AuditSuccess> #{msg}") if $log
-    end
+module Vmdb
+  module Loggers
+    class AuditLogger < VMDBLogger
+      def success(msg)
+        info("Success") { msg }
+        $log.info("<AuditSuccess> #{msg}") if $log
+      end
 
-    def failure(msg)
-      warn("Failure") { msg }
-      $log.warn("<AuditFailure> #{msg}") if $log
+      def failure(msg)
+        warn("Failure") { msg }
+        $log.warn("<AuditFailure> #{msg}") if $log
+      end
```

Comment:
Can you recreate it in console and provide a backtrace?  I have seen similar problems where constant lookup changes when you use the shortened notation but I don't want to change things without know exactly why.